test/src/userobjects/ToggleMeshAdaptivity.C

```diff
@@ -52,5 +58,6 @@ ToggleMeshAdaptivity::initialSetup()
 void
 ToggleMeshAdaptivity::execute()
 {
-  checkState();
+  if (_t_step > _steps_to_wait)
```

Comment:
Should be ``>=`` to allow waiting for zero steps.

Reply:
Yeah probably, It's just a test object though so no real impact of choosing one method over another.